.github/workflows/sonar.yml

```diff
@@ -4,39 +4,45 @@ on:
   push:
     branches:
       - main
-      - user/markackert/sonar
   pull_request:
     branches:
-      - "**"
+      - '**'
+
+permissions: read-all
 
 jobs:
   sonar:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0 
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js LTS
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
 
-    - name: Use Node.js LTS
-      uses: actions/setup-node@v2
-      with:
-        node-version: 16
+      - name: Use NPM v8
+        id: npm8
+        run: npm install -g npm@^8
```

Comment:
I believe node16 comes with npm 8 by default now.
is this still needed?

Reply:
Nope - removed it